.github/codeowners

```diff
@@ -1,2 +1,2 @@
 # Setting default codeowners to default as reviewers
-*   @silvercar/mobile-developers @silvercar/kinetic
+*   @silvercar/mobile @silvercar/kinetic
```

Comment:
The kinetic team will go away, now would be a good time to remove it.

Reply:
Force push the removal of Kinetic.